phpunit.xml

```diff
@@ -1,14 +1,14 @@
 <?xml version="1.0"?>
-<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" bootstrap="vendor/silverstripe/cms/tests/bootstrap.php" colors="true" xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.3/phpunit.xsd">
-  <coverage includeUncoveredFiles="true">
-    <include>
-      <directory suffix=".php">src/</directory>
-    </include>
-    <exclude>
-      <directory suffix=".php">tests/</directory>
-    </exclude>
-  </coverage>
-  <testsuite name="silverstripeltd-smime-forms">
-    <directory>tests/</directory>
-  </testsuite>
+<phpunit bootstrap="vendor/silverstripe/framework/tests/bootstrap.php">
```

Comment:
Initially thought the Github actions issue was at this level, looks like it wasn't but made a bunch of tidy-ups anyway to match modern standards